lib/workers/repository/process/lookup/index.ts

```diff
@@ -153,6 +153,10 @@ export async function lookupUpdates(
         latestVersion,
         allVersions.map((v) => v.version)
       );
+    // istanbul ignore if
+    if (!currentVersion) {
```

Comment:
I don't have time to work out a test for this. It only occurs because of a versioning problem which is even harder to fix.

Reply:
Does it work with docker digest updates? Current version can be undefined when no tag is used, so only image name and digest is found.